src/main/drivers/bus_spi_ll.c

```diff
@@ -24,6 +24,8 @@
 
 #include "platform.h"
 
+#include "build/atomic.h"
```

Comment:
This seems to be unneeded now. Same for other uses below, and for `drivers/nvic.h`.

Reply:
Good spot. I moved `spiSequence()` to `bus_spi.c` so all `ATOMIC_BLOCK()` calls are now there.